libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -1690,24 +1690,39 @@
         "comment": "",
         "limit": 0
     },
-    "styling.editor.dialog.name": {
+    "stylingEditor.dialog.name": {
         "value": "Name",
         "comment": "",
         "limit": 0
     },
-    "styling.editor.dialog.definition": {
+    "stylingEditor.dialog.definition": {
         "value": "Definition",
         "comment": "Name of the text field to describe an object.",
         "limit": 0
     },
-    "styling.editor.dialog.examples": {
+    "stylingEditor.dialog.examples": {
         "value": "Examples",
         "comment": "",
         "limit": 0
     },
-    "styling.editor.dialog.example.paste": {
+    "stylingEditor.dialog.example.paste": {
         "value": "Paste",
         "comment": "Button text to paste content into text field.",
         "limit": 0
+    },
+    "stylingPicker.item.edit": {
+        "value": "Edit",
+        "comment": "",
+        "limit": 0
+    },
+    "stylingPicker.item.delete": {
+        "value": "Delete",
```

Comment:
I would expect there already is something like `action.delete` translation. Please check if there is something that could be used instead of adding a new one. Maybe even for `Edit`

Reply:
There are no such strings already defined in sdk ui translations